components/CopyCodeBlock/copyCodeBlock.module.css

```diff
@@ -0,0 +1,51 @@
+/* Styles for the button before being clicked */
+
+.codeCopyBtn {
+	color: white;
+	position: absolute;
+	right: 5px;
+	top: 5px;
+	font-size: 1rem;
+	cursor: pointer;
+	transition: all 0.3s ease-in-out;
+}
+
+.codeCopyBtn:hover {
+	transform: scale(1.1);
+	opacity: 0.9;
+}
+
+.codeCopyBtn:hover::after {
```

Comment:
Is it possible for the copy/copied text to be present on the button all the time? I'd much prefer to have a button with icon and text next to each other where the text is 1rem even if the button is a little bigger. That also makes it easier for people to press.